hook/iptables.go

```diff
@@ -0,0 +1,206 @@
+package hook
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/Devatoria/go-mesos-executor/container"
+	"github.com/Devatoria/go-mesos-executor/logger"
+	"github.com/Devatoria/go-mesos-executor/types"
+	"github.com/coreos/go-iptables/iptables"
```

Comment:
Line jump before (to keep project internal packages and external packages split)